cli/tests/snapshot/inputs/pretty/simple_record.ncl

```diff
@@ -3,6 +3,6 @@
 { 
   a | Number | default = 1, 
   b : String | force = "some long string that goes past the 80 character line limit for pretty printing", 
-  c : { x : Number, y: Number } = { x = 999.8979, y = 500 },
+  c | { x : Number, y: Number } = { x = 999.8979, y = 500 },
```

Comment:
Do we want to be replacing existing tests, or adding new ones?

Reply:
I made that change thinking originally that a type signature here wouldn't work and `pprint-ast` just didn't evaluate and so doesn't notice. That's nonsense, of course. I'll revert this.

Regarding adding new tests for this change, there are a few round-trip tests in `core/src/typ.rs` that make sure the `Display` implementation for `Type` doesn't diverge far from what we expected.